### PR TITLE
#47 - removeImage Mojo added

### PR DESCRIPTION
**Pull request by krystiannowak**

### CODE REVIEW

File: src/main/java/com/spotify/docker/Utils.java [hunk 1]
```diff
@@ -64,5 +66,19 @@ public static void pushImage(DockerClient docker, String imageName, Log log)
       log.info("Pushing " + imageName);
       docker.push(imageName, new AnsiProgressHandler());
   }
+  
+  public static void removeImage(DockerClient docker, String imageName, Log log)
```
Comment (davidxia) [inline src/main/java/com/spotify/docker/Utils.java]: Maybe add a javadoc or comment here to say that `removeImage(_, true, false)` will force image removal and not delete untagged parents?

**Comment by davidxia** [comment]
@krystiannowak Nice. We should also mention how to use this with an example in the README's Usage section.

**Comment by krystiannowak** [comment]
@davidxia both usage modified and put some additional comments on params around client call

**Comment by krystiannowak** [comment]
@davidxia - do you think it's acceptable for now to merge and release it in a new version?

File: src/main/java/com/spotify/docker/Utils.java [hunk 2]
```diff
@@ -64,5 +66,21 @@ public static void pushImage(DockerClient docker, String imageName, Log log)
...
+      throws DockerException, InterruptedException {
+      log.info("Removing -f " + imageName);
+      try {
+          // force the image to be removed but it don't remove untagged parents
```
Comment (davidxia) [inline src/main/java/com/spotify/docker/Utils.java]: Can we change "it don't remove" to "it doesn't remove"?

Comment (krystiannowak) [inline src/main/java/com/spotify/docker/Utils.java]: should be in fact _don't remove_, as previously was planned to be _won't_ - but it got mixed somehow... - fix is pushed already

**Comment by davidxia** [comment]
@krystiannowak great thanks. Can you also take a look at my comment here https://github.com/spotify/docker-maven-plugin/pull/62#discussion_r26771743 and squash everything into 1 commit?

File: README.md [hunk 3]
```diff
@@ -109,6 +109,10 @@ will need to do this binding so the image gets built when maven is run from the
       </executions>
     </plugin>
 
+To remove the image run the following command:
+
+	mvn docker:removeImage
```
Comment (davidxia) [inline README.md]: @krystiannowak The user is required to supply the image name here right? eg `-DimageName=foobar`?

**Comment by krystiannowak** [comment]
@davidxia now it's all in one commit

**Comment by krystiannowak** [comment]
@davidxia thanks! now can't wait for 0.2.5 :-)

**Pull request merged**